environments/elk-ror/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
         ROR_ES_VERSION: $ROR_ES_VERSION
     ports:
       - "9200:9200"
-      - "5000:5000"
+      - "5005:5005"
```

Comment:
Why do we need this change?

Reply:
The 5000 port is taken on MacOS (I've upgraded recently :P). See [this](https://stackoverflow.com/questions/72369320/why-always-something-is-running-at-port-5000-on-my-mac)